### ISSUE TITLE
Issue. Megalodon  (acidental hit)

### ISSUE THREAD

**Issue by PeterManize** [cadon/ARKStatsExtractor#337]
I starved tamed a 145 megalodon with prime meat, although before taming it i accidentally shot it again with a tranq arrow.

Which indeed affected the taming effectiveness, but not sure to how much.
I made an estimate, and tried to mess with the effectiveness value to no fail.

The extractor only encounters an occurrence where the megalodon is "supposed" to be a level 80 tamed, with some points and a few levels already put into certain and various stats.

There wasn't any server restart. The picture I sent with the megalodon stats, i want to reveal that i have already put some points into health, since it's not a precious tame, more like "if something bad happens, its gone no beggie"

I tamed another 150 megalodon, same method. But i didn't accidentally hit it.
I made sure the effectiveness is the same using dododex, and the extractor accepted the correct stat 
![untitled](https://user-images.githubusercontent.com/29291128/26954206-a4df5964-4ca6-11e7-94c4-9a7a5e9978c8.png)
values.

**Comment by Maudor** [comment]
Hi! I have the same problem... Yesterday i tamed a megalodon with prime meat but extractor dont find the the food and damage value... Maybe value are changed?

**Comment by PeterManize** [comment]
I think the issue is regarding the flotation of values in-game, some creatures receive more bonus than others, other's don't - after tamed - then there is the spent points from the bonus levels due to taming effectiveness.

Maybe the variables aren't 100% correct mainly in the app itself. Since some times it accepts the values of the stats, although it shows incorrect information like i mentioned it:

"I starved tamed a 145 megalodon with prime meat, although before taming it i accidentally shot it again with a tranq arrow.

Which indeed affected the taming effectiveness, but not sure to how much.
I made an estimate, and tried to mess with the effectiveness value to no fail.

The extractor only encounters an occurrence where the megalodon is "supposed" to be a level 80 tamed, with some points and a few levels already put into certain and various stats."

So basically there might need a small patch or fix regarding values of some creatures, in certain levels and taming effectiveness. I have registered dilos, trikes, microraptors, iguanodons, pachys, stegos, dimorphs and many other creatures without any issue, even when taming effectiveness is crap-baskets and messed up due to honest mistakes.

**Comment by cadon** [comment]
Hi, thanks for the report. The issue is caused by rounding-errors and will be fixed in the next release.

The real taming-effectiveness is something like 70.344 %, when the tool expected at least 70.345 %, resulting some values to be wrong by just a fraction. I improved the handling of rounding-errors. Until then you can just uncheck "Consider Wild level steps" in the settings, it should work then as well.

**Comment by PeterManize** [comment]
Thanks. Glad I could help, i do really like this app a lot.
